deployment/build.centos.all

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#= CentOS 7 all .rpm
```

Comment:
So it looks like the new plan is both a clean docker build and a native build script for each distribution?

Reply:
Yes, both are possible.